theforeman.org/yaml/includes/foreman_versions_copr.yaml.inc

```diff
@@ -1 +1 @@
-[]
+- '3.9'
```

Comment:
I'd have expected this line to add this:
https://github.com/theforeman/jenkins-jobs/blob/b993393b01c86082cefda9cfea5dfbba5abd0d3c/branch-foreman#L24